Common/src/geometry_structure.cpp

```diff
@@ -9017,6 +9069,131 @@ void CPhysicalGeometry::SetPositive_ZArea(CConfig *config) {
   
 }
 
+void CPhysicalGeometry::SetHTP_Incidence(CConfig *config) {
```

Comment:
Why is this entire routine commented? Is this necessary any longer given the version in CSurfaceMovement?